### ISSUE TITLE
Progress on DataTable/DataGrid for WinUI 3

### ISSUE THREAD

**Issue by Njoro007** [microsoft/microsoft-ui-xaml#10165]
I am currently consulting on the progress of DataTable/DataGrid for WinUI 3. Coming from a WinForms background, I have extensive experience using DataGrid in WinForms applications. My primary concern is the implementation and functionality of DataTable in WinUI 3.

Could anyone provide updates or insights on the current state of DataTable/DataGrid in WinUI 3? Specifically, I am interested in:

Current capabilities and limitations

Best practices for migrating from WinForms DataGrid to WinUI 3 DataGrid

Any available resources or documentation that could assist in this transition

**Comment by castorix** [comment]
There is [DataGrid](https://learn.microsoft.com/en-us/dotnet/api/communitytoolkit.winui.ui.controls.datagrid?view=win-comm-toolkit-dotnet-7.0
)

The [doc/samples](https://learn.microsoft.com/en-us/windows/communitytoolkit/controls/datagrid) are marked UWP but they work on WinUI 3 (I did a few tests with Windows App SDK 1.6.240829007)

**Comment by lostluna1** [comment]
you need WinUi TableView on github , in WinUi3 ,no datagrid, no datatable

**Comment by Ccarmichael92** [comment]
> There is [DataGrid](https://learn.microsoft.com/en-us/dotnet/api/communitytoolkit.winui.ui.controls.datagrid?view=win-comm-toolkit-dotnet-7.0)
> 
> The [doc/samples](https://learn.microsoft.com/en-us/windows/communitytoolkit/controls/datagrid) are marked UWP but they work on WinUI 3 (I did a few tests with Windows App SDK 1.6.240829007)

It’s deprecated.

**Comment by castorix** [comment]
> > There is [DataGrid](https://learn.microsoft.com/en-us/dotnet/api/communitytoolkit.winui.ui.controls.datagrid?view=win-comm-toolkit-dotnet-7.0)
> > The [doc/samples](https://learn.microsoft.com/en-us/windows/communitytoolkit/controls/datagrid) are marked UWP but they work on WinUI 3 (I did a few tests with Windows App SDK 1.6.240829007)
> 
> It’s deprecated.

They say it has been isolated, from https://github.com/CommunityToolkit/Windows/discussions/228 : 
"Yes, DataGrid is isolated and should continue to work as the 7.x version alongside any other 8.x components."

**Comment by w-ahmad** [comment]
> you need WinUi TableView on github , in WinUi3 ,no datagrid, no datatable

You mean [WinUI.TableView](https://github.com/w-ahmad/WinUI.TableView)

**Comment by lostluna1** [comment]
> > 你需要 WinUi TableView 在 github 上，在 WinUi3 中，没有 datagrid，没有 datatable
> 
> 您指的是 [WinUI.TableView](https://github.com/w-ahmad/WinUI.TableView)



> > you need WinUi TableView on github , in WinUi3 ,no datagrid, no datatable
> 
> You mean [WinUI.TableView](https://github.com/w-ahmad/WinUI.TableView)

Yes,i think [WinUI.TableView](https://github.com/w-ahmad/WinUI.TableView) can be considered a useful Data Grid in WinUI3, while the standard Data Grid in WinUI only appears to be usable
